src/main/java/teammates/logic/core/FeedbackQuestionsLogic.java

```diff
@@ -438,8 +428,23 @@ public int compare(FeedbackQuestionAttributes q1, FeedbackQuestionAttributes q2)
         }
         return recipients;
     }
+
+    private String getGiverTeam(String defaultTeam, InstructorAttributes instructorGiver,
+            StudentAttributes studentGiver) {
+        String giverTeam = null;
+        boolean isStudentGiver = studentGiver != null;
+        boolean isInstructorGiver = instructorGiver != null;
+        if (isStudentGiver) {
+            giverTeam = studentGiver.team;
+        } else if (isInstructorGiver) {
+            giverTeam = Const.USER_TEAM_FOR_INSTRUCTOR;
+        } else {
+            giverTeam = defaultTeam;
+        }
```

Comment:
Sorry I missed this earlier, but since this else statement sets a default `giverTeam` anyway, we could just shift this to the initialisation of `giverTeam`, and remove the need for this else statement.